search-service/src/test/kotlin/com/egm/stellio/search/service/EntityOperationServiceTests.kt

```diff
@@ -150,53 +153,72 @@ class EntityOperationServiceTests {
             listOf(BatchEntityError(secondEntityURI, arrayListOf("error"))),
             batchOperationResult.errors
         )
+        coVerify {
+            entityEventService.publishAttributeChangeEvents(
+                sub,
+                eq(firstEntityURI),
+                any(),
+                match { it.isSuccessful() },
+                any()
+            )
+        }
+        coVerify {
+            entityEventService.publishAttributeChangeEvents(
+                sub,
+                eq(secondEntityURI),
+                any(),
+                any(),
+                any()
+            ) wasNot Called
+        }
     }
 
     @Test
-    fun `processEntities should count as error not processed attributes in entities`() =
-        runTest {
-            val updateResult = UpdateResult(
-                emptyList(),
-                listOf(
-                    NotUpdatedDetails("attribute#1", "reason"),
-                    NotUpdatedDetails("attribute#2", "reason")
-                )
+    fun `processEntities should count as error not processed attributes in entities`() = runTest {
```

Comment:
the name of the test is a bit confusing. I'm not sure I understand what it should do

Reply:
it was alredy there before, not adding unnecessary modifications